composer.json

```diff
@@ -17,6 +17,8 @@
     "require": {
         "php": "^7.4|^8.0",
         "spatie/laravel-permission": "^4.0",
+        "vyuldashev/nova-permission": "^2.11",
+        "laravel/cashier": "^12.6",
```

Comment:
@drewroberts I added them here, but I think you will move to the support package, just to keep in mind

Reply:
Maybe. This is good for now.